boards/arm/google_twinkie_v2/doc/index.rst

```diff
@@ -0,0 +1,59 @@
+.. _google_twinkie_v2_board:
+
+Google Twinkie V2
+#################
+
+Overview
+********
+
+Google Twinkie V2 is a reference board for the google power delivery analyzer
+(PDA) Twinkie V2.
+
+Zephyr has support for the STM32-based embedded controller (EC) on-board.
```

Comment:
Don't think it's right to call it an EC in this case, the "EC" term is somewhat specific of the support chip in laptops, it's just the MCU that the board runs. Also it's fairly given since one person is reading this in the Zephyr documentation. :-) I'd say just drop this line.